apps/src/lib/kits/maker/ui/SetupStep.jsx

```diff
@@ -8,7 +8,8 @@ export const ATTEMPTING = 'ATTEMPTING';
 export const SUCCEEDED = 'SUCCEEDED';
 export const FAILED = 'FAILED';
 export const CELEBRATING = 'CELEBRATING';
-export const STEP_STATUSES = [HIDDEN, WAITING, ATTEMPTING, SUCCEEDED, FAILED, CELEBRATING];
+export const UNKNOWN = 'UNKNOWN';
+export const STEP_STATUSES = [HIDDEN, WAITING, ATTEMPTING, SUCCEEDED, FAILED, CELEBRATING, UNKNOWN];
```

Comment:
I wonder if instead of a bunch of individual consts, this should be an enum, i.e.
```
const SetupStatus = {
  HIDDEN: 'HIDDEN',
  // etc.
};
```
This also means that instead of needing a separate array of possible statuses, we can just do `Object.values(SetupStatus)`

Another pattern I've followed elsewhere is to then stick the enum on the Component: `SetupStep.SetupStatus = SetupStatus`. This allows consumers to do something like
```
<SetupStep stepStatus={SetupStep.SetupStatus.HIDDEN}/>
```

cc @islemaster

Reply:
I don't have strong feelings about this.  I like the individual exports because I think it's concise and, in most cases, a consumer will have a narrow enough scope that
```jsx
import SetupStep, {HIDDEN} from './SetupStep';
<SetupStep stepStatus={HIDDEN}/>
```
is just as clear as
```jsx
import SetupStep from './SetupStep';
<SetupStep stepStatus={SetupStep.SetupStatus.HIDDEN}/>
```
If you have a larger consumer where `HIDDEN` might be ambiguous, agreed that the latter is helpful, but you can always `import SetupStep, * as SetupStatus from './SetupStep';` if needed.

Reply:
It feels a little awkward to me to need to have a separate array of STEP_STATUSES.